draft-ietf-quic-transport.md

```diff
@@ -3338,6 +3342,52 @@ then a packet containing a 16-bit value of 0x9b32 will be decoded as 0xa82f9b32.
 Example pseudo-code for packet number decoding can be found in
 {{sample-packet-number-decoding}}.
 
+### Starting Packet Numbers
```

Comment:
I don't think that we need this still, but absent analysis, this can be moved under the Initial packet section.